inc/Engine/Optimization/DynamicLists/DynamicLists.php

```diff
@@ -232,4 +235,45 @@ public function get_combine_js_excluded_inline(): array {
 
 		return isset( $lists->js_excluded_inline ) ? $lists->js_excluded_inline : [];
 	}
+
+	/**
+	 * Get list of exclusions from the API list.
+	 *
+	 * @param string $item_id Item ID to get exclusions for (plugin slug, theme slug, script ID).
+	 *
+	 * @return array
+	 */
+	public function get_delayjs_exclusions_by_id( string $item_id ) {
```

Comment:
This function is not used anywhere in the code

Reply:
@engahmeds3ed will add the part where it is used. (I left it there as I was thinking this part was a part of the last issue and that seemed total waste of time to drop it to readd it later)